### ISSUE TITLE
Use Sentry.io for error reporting

### ISSUE THREAD

**Issue by mscottford** [corgibytes/freshli#117]
See docs for more info: https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/

**Comment by dan-hein** [comment]
Is this something that we want to integrate with Freshli-Web as opposed to Freshli-Core?

**Comment by mscottford** [comment]
Yes. This issue was created before the repositories were split apart. This would make the most sense to implement in the freshli-web and freshli-cli projects.

M. Scott Ford
Co-Founder & CEO
Corgibytes, LLC
804.596.2375 x701
pronouns: he/him
scott@corgibytes.com
https://corgibytes.com ( https://corgibytes.com/ )

Have you read our First Round Review ( http://firstround.com/review/forget-technical-debt-heres-how-to-build-technical-wealth/ ) article about paying off technical debt?

Love refactoring and TDD? Join us at LegacyCode.Rocks ( http://LegacyCode.Rocks ) for virtual meetups, podcasts, and more.

Sent via Superhuman iOS ( https://sprh.mn/?vip=scott@corgibytes.com )
